test/test-helper.ts

```diff
@@ -8,6 +8,39 @@ const path = require("path");
 const fs = require("fs");
 const fse = require("fs-extra");
 
+class Query {
```

Comment:
Maybe now rename to `QueryableStack` or `TestStack` or similar as this is not the result of a `$()` call anymore?